packages/driver/src/cross-origin/origin_fn.ts

```diff
@@ -66,6 +61,38 @@ const rehydrateRunnable = (data: serializedRunnable): Runnable|Test => {
   return runnable
 }
 
+const getCallbackFn = async (fn: string, file?: string) => {
+  if (
+    // @ts-ignore
+    !Cypress.config('experimentalOriginDependencies')
+    || !fn.includes('Cypress.require')
+  ) {
+    return fn
+  }
+
+  const callbackName = '__cypressCallback'
+  const response = await fetch('/__cypress/process-origin-callback', {
```

Comment:
I think this happens after `fetch` gets patched in the spec bridge, but it shouldn't matter here I think unless there is some reason we need to side step it